artifactory_cleanup/rules/docker.py

```diff
@@ -19,6 +19,7 @@ class RuleForDocker(Rule):
     """
 
     MANIFEST_FILENAME = "manifest.json"
+    BATCH_SIZE = 40
```

Comment:
Is there way to have it in config with **40** by default if not provided for backward compatability 🤔 

Why 40 btw, let's do 42 may be? 😆 

Just kidding, let's change to **100**, idk, something that doesn't look like magic